### DISCUSSION TITLE
Polling time

### DISCUSSION THREAD

**Discussion by PeterPedro** [Tkd-Alex/Twitch-Channel-Points-Miner-v2#249] [Ideas]
How often does it poll twitch to see if the streamer is streaming? I think it would be nice to be able to change the polling time as a setting.

**Comment by Rakambda** [comment]
Status is updated by events received 
https://github.com/Tkd-Alex/Twitch-Channel-Points-Miner-v2/blob/809c4c9e7e0cba832ac32deea1da87110478b11b/TwitchChannelPointsMiner/classes/WebSocketsPool.py#L212

Otherwise it is checked every ~10 minutes 
https://github.com/Tkd-Alex/Twitch-Channel-Points-Miner-v2/blob/809c4c9e7e0cba832ac32deea1da87110478b11b/TwitchChannelPointsMiner/classes/Twitch.py#L257
